CMakeLists.txt

```diff
@@ -590,17 +593,24 @@ if(CIRCT_SLANG_FRONTEND_ENABLED)
     # statically link slang into the CIRCTImportVerilog library, but seems to be
     # harder than it ought to be.
     set_property(
```

Comment:
Yeah, it seems like all this install / export stuff shouldn't be needed but I'm not enough of a CMake expert to know offhand how to fix it.